students/hirot_directory/session05/mailroom5.py

```diff
@@ -29,7 +33,7 @@ def print_list():
                                            msg4="Average Gift"))
     for d in donor_dict:
         t = sum(donor_dict[d])
```

Comment:
when looping thourhg a dict, if you find you need both the key and the value, you can do:

```
for donor, donations in donr_dict.items():
    t = sum(donations)
    n = len(donations)
   ...
```